ui/common/src/socket.ts

```diff
@@ -196,21 +192,17 @@ export default class StrongSocket {
       document.body.classList.add('offline');
       document.body.classList.remove('online');
       $('#network-status').text(site.trans ? site.trans('reconnecting') : 'Reconnecting');
-      if (!this.tryOtherUrl && isOnline()) {
-        // if this was set earlier, we've already logged the error
-        this.tryOtherUrl = true;
-        log(`sri ${this.settings.params!.sri} timeout ${delay}ms, trying ${this.baseUrl()}${this.url}`);
-      }
```

Comment:
the socket permalogs are noisy and mostly serve to obscure some of the more interesting uncaught exceptions and unhandled promise rejections we get from lichess.org/#debug